Sources/StreamChatUI/ChatMessageList/Attachments/MixedAttachmentViewInjector.swift

```diff
@@ -3,15 +3,52 @@
 //
 
 import Foundation
+import StreamChat
 
 /// The injector used to combine multiple types of attachment views.
 open class MixedAttachmentViewInjector: AttachmentViewInjector {
-    open lazy var injectors = [
-        contentView.components.galleryAttachmentInjector.init(contentView),
-        contentView.components.filesAttachmentInjector.init(contentView),
-        contentView.components.voiceRecordingAttachmentInjector.init(contentView)
+    /// The registry of injectors associated with their attachment type, that support mixed attachment rendering.
+    ///
+    /// **Note:** It is an array and not a dictionary, since it defines the order of the rendering of the different types of attachments.
+    /// In order to customise the order, this static property can be changed.
+    ///
+    /// By default, this is the order of how mixed attachments are rendered:
+    ///
+    ///     1. Images and Videos
```

Comment:
Looking at the code, it's videos then images.

Reply:
This is fine, the order is by the Injector, the type does not really matter here 👍